apps/modernization-ui/src/SkipLink/SkipLinkContext.tsx

```diff
@@ -24,14 +26,30 @@ interface SkipLinkProviderProps {
 
 export const SkipLinkProvider = ({ children }: SkipLinkProviderProps) => {
     const [currentFocusTarget, setCurrentFocusTarget] = useState('');
+    const [previousFocusTarget, setPreviousFocusTarget] = useState('');
```

Comment:
question:  How is `currentFocusTarget` different from `previousFocusTarget`?  It seems to be the same thing here